scripts/tutorial-patches/RocketConfigs.scala.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/generators/chipyard/src/main/scala/config/RocketConfigs.scala b/generators/chipyard/src/main/scala/config/RocketConfigs.scala
+index f0548b2..b0d0135 100644
+--- a/generators/chipyard/src/main/scala/config/RocketConfigs.scala
++++ b/generators/chipyard/src/main/scala/config/RocketConfigs.scala
+@@ -293,7 +293,7 @@ class Sha3RocketConfig extends Config(
+   new chipyard.config.WithBootROM ++
+   new chipyard.config.WithUART ++
+   new chipyard.config.WithL2TLBs(1024) ++
+-  new sha3.WithSha3Accel ++                                // add SHA3 rocc accelerator
++//  new sha3.WithSha3Accel ++                                // add SHA3 rocc accelerator
```

Comment:
can we just make this a different config with the commented out bit, rather than modifying this?

Reply:
like Sha3RocketTutorialConfig,
which has this same line commented out

Reply:
Note that we need a checked in config with the fragment enabled for regressions, see the new `chipyard-sha3` tests which verify the Sha3 thing working

Reply:
Wait what the TutorialConfigs exist in TutorialConfigs.scala. My point above was that we still need a config with the accel built in to enable CI on the accel